src/dndbeyond/content-scripts/encounter.js

```diff
@@ -26,20 +26,24 @@ function documentModified(mutations, observer) {
 }
 
 function updateCombatTracker() {
-    if ($(".turn-controls__next-turn-button").length) {
-        $(".ui-dialog-buttonpane").hide(); // get rid of turn controls, to make it more obvious to do things from DDB.
-        const combat = Array.from($(".combatant-card.in-combat")).map(x => [
-            $(".combatant-summary__name", x).text(),
-            $(".combatant-card__initiative-value", x).text(),
-            $(x).hasClass("is-turn"),
-        ]);
-        const req = {
-            action: "combat-tracker",
-            combat,
-        };
-        console.log("Sending combat update", combat);
-        chrome.runtime.sendMessage(req, resp => beyond20SendMessageFailure(character, resp));
+    if (!$(".turn-controls__next-turn-button").length) {
+        return;
```

Comment:
oufff, I don't like one liners that use accolades.. I'll let it slide though...

Reply:
Took me a bit to work out what you meant by accolades - but here's some light reading: https://medium.com/@jonathanabrams/single-line-if-statements-2565c62ff492

Reply:
Oh sorry, 'accolades' is the french term, I forget it's called braces in english.
And yep, I'm aware of the risk of it and the possibility of making mistakes because of that. It's a personal choice though, I really don't like seeing one liners with the braces because visually it feels too heavy. For early return like that with the single return statement, I'd have put it on the same line as well (I also used to hate if statements with their statement on the same line but gotten used to it recently for short return statements only).

I'm used to it though, so I don't fall into the trap of editing code and forgetting to add the braces.
Like I said, I'll let it slide though. :p